src/evd/classic/evd.cpp

```diff
@@ -1,7 +1,7 @@
 #include "evd.hpp"
 #include <math.h>
+#include <stdio.h>
```

Comment:
Remove this. I guess `stdio.h` is not necessary.

Reply:
Done!